docs/src/modules/utils/generateMarkdown.js

```diff
@@ -244,7 +244,7 @@ function generateProps(reactAPI) {
       return textProps;
     }
 
-    let defaultValue = '\u00a0';
+    let defaultValue = '';
```

Comment:
Shouldn't this be 
```suggestion
    let defaultValue = '&nbsp;';
```
?

Reply:
Haha! I was looking at the change history. I have no clue why we render an empty space in the first place. I wish I used atomic commits.

Reply:
What do you think of tryig without?

Reply:
I could only look at this in isolation. If you say this is fine without then we can go with that.

Reply:
I don't know if it's fine. I couldn't spot any regression. I would like to take a chance :).